cmd/gogio/main.go

```diff
@@ -34,7 +34,9 @@ var (
 	linkMode      = flag.String("linkmode", "", "set the -linkmode flag of the go tool")
 	extraLdflags  = flag.String("ldflags", "", "extra flags to the Go linker")
 	extraTags     = flag.String("tags", "", "extra tags to the Go tool")
-	iconPath      = flag.String("icon", "", "Specify an icon for iOS and Android")
+	iconPath      = flag.String("icon", "", "specify an icon for iOS and Android")
+	signKey       = flag.String("key", "", "specify the path of the keystore to be used to sign binaries.")
```

Comment:
Let's call it "signkey" to  emphasize the purpose. Add "Android" in the help text to make it clearer it only applied there.